src/router.js

```diff
@@ -62,17 +53,14 @@ const makeMetaTags = (defaultTitle) => ({
 });
 
 /* Routing mode, can be either 'hash', 'history' or 'abstract' */
-const mode = appConfig.routingMode || 'history';
+const mode = window.extraConf.routingMode || 'history';
 
 /* List of all routes, props, components and metadata */
 const router = new Router({
   mode,
   routes: [
-    { // The default view can be customized by the user
+    {
       path: '/',
-      name: `landing-page-${getStartingView()}`,
-      component: getStartingComponent(),
-      meta: makeMetaTags('Home Page'),
     },
```

Comment:
Minor thing, but the default home route is appending `/home` onto the URL, which shouldn't be needed/ neater not to have the URL change.